### PR TITLE
(IO-2) Importing command feature

### PR DESCRIPTION
**Pull request by Matx1002**

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/command/CommandImporter.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.io.IOException;
+
+public interface CommandImporter {
+    DriverCommand importCommands(String filePath);
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/CommandImporter.java:6]: I think that filePath is a wrong abstraction. Commands are imported directly from text strings. These text strings can be read from files, textboxes, url's, etc.

File: src/main/java/edu/kis/powp/jobs2d/command/JsonCommandImporter.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package edu.kis.powp.jobs2d.command;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/JsonCommandImporter.java:8]: These dependencies are not necessary for JsonCommandImporter

File: src/main/java/edu/kis/powp/jobs2d/command/JsonCommandImporter.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import java.util.logging.Logger;
+
+public class JsonCommandImporter implements CommandImporter {
+
+    private final static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+
+    @Override
+    public DriverCommand importCommands(String filePath) {
+        try {
+            String content = new String(Files.readAllBytes(Paths.get(filePath)));
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/JsonCommandImporter.java:20]: unnecessary additional responsibility (SRP). The source of "content" aspect should be OCP.

File: src/test/java/edu/kis/powp/jobs2d/figures/rectangle.json
Comment (kowallus) [inline src/test/java/edu/kis/powp/jobs2d/figures/rectangle.json:1]: these files should be located in test resources folder

**Review by kowallus — CHANGES_REQUESTED** [review]
Please, more SRP and OCP.

**Comment by Matx1002** [comment]
Changes have been made. Please re-review

**Comment by Matx1002** [comment]
Ok, done. Please re-review.

**Review by kowallus — CHANGES_REQUESTED** [review]
Dependency to JsonCommandImporter not removed from the factory.
The factory is almost OCP (lack of configuration ability).

**Pull request merged**